Modules/Core/SpatialObjects/test/itkImageSpatialObjectTest.cxx

```diff
@@ -179,12 +187,17 @@ itkImageSpatialObjectTest(int, char *[])
   expectedDerivative[1] = 10;
   expectedDerivative[2] = 100;
   std::cout << "DerivativeAt() with interpolator ...";
-  if (itk::Math::abs(derivative[0] - expectedDerivative[0]) > 0.00001 ||
-      itk::Math::abs(derivative[1] - expectedDerivative[1]) > 0.00001 ||
-      itk::Math::abs(derivative[2] - expectedDerivative[2]) > 0.00001)
+  double epsilon = 0.00001;
```

Comment:
re-declaration (s. line 168)

Reply:
Thanks. Had already spot that. Addressed in ff46ef4.